infrastructure/zk/src/run/forced-exit.ts

```diff
@@ -0,0 +1,129 @@
+import { Command } from 'commander';
+import fetch from 'node-fetch';
+import * as utils from '../utils';
+import { BigNumber, BigNumberish, ethers } from 'ethers';
+
+type State = {
+    balances: {
+        [token: string]: BigNumberish;
+    };
+    nonce: number;
+    pubKeyHash: string;
+};
+
+type RestAccountState = {
+    finalized: State | undefined;
+};
+
+type RpcAccountState = {
+    verified: State;
+};
+
+async function checkForcedExitSenderAccountBalance(nodeUrl: string, nodeType: 'REST' | 'JSONRPC') {
+    const forcedExitAccount = process.env.FORCED_EXIT_REQUESTS_SENDER_ACCOUNT_ADDRESS as string;
+    let state: State | undefined;
+    if (nodeType === 'REST') {
+        ({ finalized: state } = await processRestRequest(nodeUrl, forcedExitAccount));
+    } else {
+        ({ verified: state } = await processJsonRpcRequest(nodeUrl, forcedExitAccount));
+    }
```

Comment:
```suggestion
const state = nodeType === 'REST' ? 
        (await processRestRequest(nodeUrl, forcedExitAccount)).finalized
        : (await processJsonRpcRequest(nodeUrl, forcedExitAccount)).verified;
```

Reply:
updated

Reply:
```suggestion
const state = (await processRestRequest(nodeUrl, forcedExitAccount))[nodeType === 'REST' ? 'finalized' : 'verified'];
```
just saying ... ;)

Reply:
its not a valid suggestion, I tried to apply it but it gave me errors :S

Reply:
I guess it's not valid because in one branch of the `if` statement we call `processJsonRequest` (note "...**Json**...") while in the other we call `processRestRequest` (note "...**Rest**...")